myning/migrations/black_smith_upgrade.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+from myning.config import UPGRADES
+from myning.objects.player import Player
+from myning.utils.file_manager import FileManager
+
+
+def run():
+    with open(".data/player.json") as f:
+        player = Player()
+
+        if player.has_upgrade("buy_full_set"):
+            upgrade = UPGRADES["buy_full_set"]
+            upgrade.level = 5
```

Comment:
this is Jedi and not Blademaster?

Reply:
Yeah. If you bought it before, you spent 100k to do so. 100k gets you to around level 5 I think.

Reply:
Maybe should be level 6 actually

Reply:
level 5 is a little less in total money. and before, your upgrade was for the blademaster set so 6 makes sense to "grandfather" the legacy upgrade. although I don't think anyone has this right now but you